app/views/searches/show.html.erb

```diff
@@ -7,6 +7,17 @@
   </div>
 <% end %>
 
+<% if @gems && @gems.size < 1 && @gems.respond_to?(:response) && (suggestions = @gems.response.response['suggest']) && suggestions.present? && suggestions['suggest_name'].present? && suggestions['suggest_name'][0]['options'].length > 0  %>
+  <div class='search-suggestions'>
+    <p>
+      Maybe you mean
+      <%= suggestions.map { |k,v| v.first['options'] }.flatten.map {|v| v['text']}.uniq.map do |term|
```

Comment:
I think this should be also better supported by the gem itself — I just blindly return the ES JSON here, and it's not the ideal structure for an expressive environment like Rails...